vernac/library.ml

```diff
@@ -480,12 +489,7 @@ let save_library_to todo_proofs ~output_native_objects dir f otab =
   let ch = raw_extern_library f' in
```

Comment:
Why do we have `let f' = f in` immediately above this line?

Reply:
Yeah I did another pass; this was done to avoid diff noise.